tailormap-components/projects/core/src/lib/feature-form/form-creator/form-creator.component.css

```diff
@@ -4,13 +4,19 @@ mat-tab-group {
   height: 100%;
 }
 
+.tab {
+  float: left;
```

Comment:
Maybe better to use a display: flex parent & set flex: 1 on the tabs. Floating can break your layout quite easily and then you also don't need to set a width